src/models/db/notification.js

```diff
@@ -10,6 +11,13 @@ class Notification extends Model {
     delete data.updated_at
     return data
   }
+
+  static convertToPersistent (data) {
+    if (!data.id) {
+      data.id = uuid()
```

Comment:
I'm not sure if this is the right place for this. Could we do it wherever the notification is being created?